tests/sentry/digests/backends/test_redis.py

```diff
@@ -107,3 +107,14 @@ def test_missing_record_contents(self):
         # contents were merged back into the digest.
         with backend.digest('timeline', 0) as records:
             assert set(records) == set([record_2])
+
+    def test_large_digest(self):
+        backend = RedisBackend()
+
+        n = 16384
```

Comment:
Its just the first power of 2 that caused the error to occur on my local installation (3.2.8.) (I reran it and 8192 actually triggered the error, I'm not sure why this wasn't the case before. I can drop the value here if needed.)

From what I could tell the stack size hasn't been constant over time — in today's issue the data set was significantly smaller (on the order of 3000 items) and that caused an issue with whatever version we're running.